mod/missions/views/default/page/elements/mission-manager-info.php

```diff
@@ -14,7 +14,7 @@
 $container_class = $vars['container_class'];
 $grid_number = $vars['grid_number'];
 $test = $mission->account;
-$manager_account = get_user($mission->account); //Nick changed to owner_guid then back to account
+$manager_account = get_user($mission->owner_guid); //Nick changed to owner_guid then back to account
```

Comment:
So, I think I fix it. What I did is I compare the email and the name, if I have a match, I take the guid from this